R/ggplot2-package.R

```diff
@@ -5,6 +5,7 @@
 #' @import scales grid gtable rlang vctrs
 #' @importFrom glue glue glue_collapse
 #' @importFrom lifecycle deprecated
+#' @importFrom S7 class_any class_list method method<- new_generic new_S3_class S7_dispatch
```

Comment:
I'd be tempted to minimise the imports to make it most clear where S7 is being used (at least for now).